pyproject.toml

```diff
@@ -0,0 +1,92 @@
+[tool.poetry]
+name = "victorciurana.com"
+version = "0.1.0"
+description = ""
+authors = ["Victor <admin@victorciurana.com>"]
+
+[tool.poetry.dependencies]
+appdirs = "1.4.4"
+asgiref = "3.3.4"
+attrs = "21.2.0"
+certifi = "2020.12.5"
+chardet = "4.0.0"
+click = "7.1.2"
+Django = "3.2"
+django-markdownx = "3.0.1"
+djangorestframework = "3.12.4"
+django-otp = "1.0.5"
+gunicorn = "20.1.0"
+idna = "2.10"
+iniconfig = "1.1.1"
+Markdown = "3.3.4"
+martor = "1.6.2"
+mypy-extensions = "0.4.3"
+packaging = "20.9"
+psycopg2-binary = "2.8.6"
+pathspec = "0.8.1"
+pluggy = "0.13.1"
+py = "1.10.0"
+pyparsing = "2.4.7"
+python-environ = "0.4.54"
+pytz = "2021.1"
+qrcode = "6.1"
+regex = "2021.4.4"
+requests = "2.25.1"
+six = "1.15.0"
+sqlparse = "0.4.1"
+toml = "0.10.2"
+urllib3 = "1.26.4"
+
+[tool.poetry.dev-dependencies]
+appdirs = "1.4.4"
+asgiref = "3.3.4"
+attrs = "21.2.0"
+black = "21.4b2"
+certifi = "2020.12.5"
+chardet = "4.0.0"
+click = "7.1.2"
+distlib = "0.3.1"
+Django = "3.2"
+django-markdownx = "3.0.1"
+django-otp = "1.0.5"
+djangorestframework = "3.12.4"
```

Comment:
Your dependencies are quite messed up, for instance, `djangorestframework` is duplicated, you have it both in `dependencies` and `dev-dependencies`.

`dev-dependencies` should contain only those dependencies you need locally, like for example `pytest` or `isort`, i.e. dependencies you won't install in the server (because they are only used during development). You should clean this up.

Reply:
so the ones on ```dependencies``` also get installed while locally developing right?
I thought it was two completely different set of dependencies

Reply:
okey, I got it now